.github/workflows/build-and-publish.yml

```diff
@@ -41,9 +41,16 @@ jobs:
       - run: echo "value=$(ls dist/*.tar.gz | sed 's/.tar.gz//' | xargs basename)" >> $GITHUB_OUTPUT
         id: dist_folder
 
-      - uses: "marvinpinto/action-automatic-releases@latest"
+      - name: Publish distribution to GitHub
+        uses: "marvinpinto/action-automatic-releases@latest"
         with:
           repo_token: "${{ secrets.GITHUB_TOKEN }}"
           files: |
             dist/*.tar.gz
             dist/*.whl
+
+      - name: Publish distribution to PyPI
+        uses: pypa/gh-action-pypi-publish@master
+        with:
+          user: __token__
+          password: ${{ secrets.PYPI_API_TOKEN }}
```

Comment:
I got a token generated for this project, I can pass it to you @Stedders then see if github action works with pypi or not?

Reply:
Sounds good. Shall we connect on Saturday?

I have questions about the token. Is it associated with your account? If so I would prefer a generic one associated with a coderdojo account.

Reply:
👍 
It is associated with [this particular project](https://pypi.org/project/coder-dojo-common-python/) in my account, I just wanted to get pypi up and running first so people can actually pip install thing. 

I think the right thing to do is probably link everything back to the [coderdojo account](https://github.com/coder-dojo-ham). I'll have a look.

Reply:
Ok. That's fine. Feel free to merge. I won't be able to look at this again until Wednesday.

Reply:
Oh. You need to give me the token.

LOL